src/ProjectAudioManager.cpp

```diff
@@ -552,7 +552,8 @@ void ProjectAudioManager::Stop(bool stopStream /* = true*/)
       }
    }
 
-   const auto toolbar = ToolManager::Get( *project ).GetToolBar(ScrubbingBarID);
+   // To do: eliminate this, use an event instead
+   const auto toolbar = ToolManager::Get( *project ).GetToolBar(wxT("Scrub"));
```

Comment:
Should the GetToolBar call ScrubberToolBar::ID() instead of wxT("Scrub")?

Reply:
No:  that would make an undesirable dependency cycle.

The Scrubbing toolbar (and each other toolbar) should be easily extractible as a module.  Such a cycle would prohibit that.

Lookups by string, which may fail, are a weaker sort of dependency.

Notice the null check on toolbar and also my new comment.

A lot of old scrubbing code (which I wrote, years ago) makes some difficult tangling in the core application that I must still figure out.